source/dub/test/base.d

```diff
@@ -212,6 +223,21 @@ package class TestPackageManager : PackageManager
 
 		return null;
     }
+
+    /**
+     * Adds a `Package` to this `PackageManager`
+     *
+     * This is currently only available in unittests as it is a convenience
```

Comment:
documentation is lying, this is publicly available now and not only in unittests.

Also, using spaces instead of tabs

Reply:
There is a `version (unittest)` at the top of the module.